src/Tests/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -817,6 +817,26 @@ public void PathEndsWithSlashOrBackslash(string flag, string slashesOrBackslashe
             Directory.EnumerateFileSystemEntries(flagDirectory).Should().NotBeEmpty();
         }
 
+        [Theory]
+        [InlineData("-e:foo=bardll")]
+        [InlineData("-e:foo=barexe")]
+        public void ArgumentsEndWithDllOrExeShouldNotFail(string arg)
+        {
+            var testProjectDirectory = CopyAndRestoreVSTestDotNetCoreTestApp();
+
+            // Call test
+            CommandResult result = new DotnetTestCommand(Log)
+                .Execute(testProjectDirectory, arg);
+
+            // Verify
+            if (!TestContext.IsLocalized())
+            {
+                result.StdOut.Should().Contain("Total:     2");
+                result.StdOut.Should().Contain("Passed:     1");
+                result.StdOut.Should().Contain("Failed:     1");
```

Comment:
Could you add a comment explaining what is passed and what is failed? I don't know if we can access the details and check for the arg data.

Reply:
We have this output
```
Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
  Failed VSTestFailTest [17 ms]
  Error Message:
   Assert.Fail failed. 
  Stack Trace:
     at TestNamespace.VSTestTests.VSTestFailTest() in C:\git\sdk\artifacts\tmp\Debug\ArgumentsEndW---D9229B9D\Tests.cs:line 19
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)


Failed!  - Failed:     1, Passed:     1, Skipped:     0, Total:     2, Duration: 48 ms - VSTestCore.dll (net7.0)
```
Maybe I can add a contains for `VSTestFailTest`

Reply:
Update assertion

Reply:
Or at least just a small comment in the test assertion to say you expect failed test to be `VSTestFailTest` and passing one to be arg.